gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
-distributionSha256Sum=f6b8596b10cce501591e92f229816aa4046424f3b24d771751b06779d58c8ec4
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
distributionSha256Sum=3e1af3ae886920c3ac87f7a91f816c0c7c436f276a6eefdb3da152100fef72ae
```

Source: https://gradle.org/release-checksums/

---

done  in https://github.com/ACINQ/lightning-kmp/pull/594/commits/87d472a87e6ee6b491aca0a8899038ecac1df4c7